File: PHP/Sumisang.php
<?php
class Sumisang {
    private $code, $name, $category, $desc, $photo;

    public function __construct($code = '', $name = '', $category = '', $desc = '', $photo = '') {
        $this->code = $code;
        $this->name = $name;
        $this->category = $category;
        $this->desc = $desc;
        $this->photo = $photo;
    }

    public function getCode() { 
        return $this->code; 
    }
    public function setCode($code) { 
        $this->code = $code; 
    }
    
    public function getName() { 
        return $this->name; 
    }
    public function setName($name) { 
        $this->name = $name; 
    }

    public function getCategory() { 
        return $this->category; 
    }
    public function setCategory($category) { 
        $this->category = $category; 
    }

    public function getDesc() { 
        return $this->desc; 
    }
    public function setDesc($desc) { 
        $this->desc = $desc; 
    }

    public function getPhoto() { 
        return $this->photo; 
    }
    public function setPhoto($photo) { 
        $this->photo = $photo; 
    }
}
?>

File: PY/Sumisang.py
class Sumisang:
    def __init__(self, code="", name="", category="", desc=""):
        # Private attributes
        self.__code = code
        self.__name = name
        self.__category = category
        self.__desc = desc

    # Getter and Setter for 'code'
    def getcode(self):
        return self.__code

    def setcode(self, value):
        self.__code = value

    # Getter and Setter for 'name'
    def getname(self):
        return self.__name

    def setname(self, value):
        self.__name = value

    # Getter and Setter for 'category'
    def getcategory(self):
        return self.__category

    def setcategory(self, value):
        self.__category = value

    # Getter and Setter for 'desc'
    def getdesc(self):
        return self.__desc

    def setdesc(self, value):
        self.__desc = value
